stl/inc/vector

```diff
@@ -153,12 +153,12 @@ public:
     }
 
 #if _HAS_CXX20
-    _NODISCARD strong_ordering operator<=>(const _Vector_const_iterator& _Right) const noexcept {
+    _NODISCARD _CONSTEXPR20_CONTAINER strong_ordering operator<=>(const _Vector_const_iterator& _Right) const noexcept {
         _Compat(_Right);
         return _Unfancy(_Ptr) <=> _Unfancy(_Right._Ptr);
     }
 #else // ^^^ _HAS_CXX20 ^^^ / vvv !_HAS_CXX20 vvv
-    _NODISCARD bool operator!=(const _Vector_const_iterator& _Right) const noexcept {
+    _NODISCARD _CONSTEXPR20_CONTAINER bool operator!=(const _Vector_const_iterator& _Right) const noexcept {

```

Comment:
`_CONSTEXPR20_CONTAINER` is always simply `inline` in pre-C++20 modes. This is at best confusing - I think we should revert the change on this line. (Also on 2223.)

Reply:
STL points out that _all_ of these operators that are disabled in C++20 mode should not be `_CONSTEXPR20_CONTAINER`.

Reply:
Fixed all occurrences (I think) of `_CONSTEXPR20` and `_CONSTEXPR20_CONTAINER` appearing in `!_HAS_CXX20` mode.

There are occurrences in `<array>` and `<vector>` where `< <= > >=` (marked as `_CONSTEXPR20` and `_CONSTEXPR20_CONTAINER`) are guarded by `!defined(__cpp_lib_concepts)`; those cannot be changed yet.

`allocator` has an `operator!=` marked `_CONSTEXPR20_DYNALLOC`, but `allocator` is still on the todo list for Clause 20, not changed here.

Reply:
> There are occurrences in `<array>` and `<vector>` where `< <= > >=` (marked as `_CONSTEXPR20` and `_CONSTEXPR20_CONTAINER`) are guarded by `!defined(__cpp_lib_concepts)`; those cannot be changed yet.

Recorded [a cleanup reminder](https://github.com/microsoft/STL/issues/395#issuecomment-781809236) in #395.